test/nuntiux_test.exs

```diff
@@ -213,4 +320,10 @@ defmodule NuntiuxTest do
 
     echoer()
   end
+
+  defp safe_unregister(name) do
+    Process.unregister(name)
+  rescue
+    _error -> :ok
+  end
```

Comment:
If one test failed many other might also fail, because `name` would still be registered. This way we force unregister at start and stop (kinda think that doing it `on_exit` is redundant, though.